javascript/finishing_up_with_javascript/conclusion.md

```diff
@@ -1,32 +1,15 @@
 ### The end of the JavaScript course! 
-Completing the JavaScript course is a major milestone! Congratulations! 
 
-At this point, you can create phenomenal and dynamic frontends. These will integrate with the back-ends The Odin Project teaches, Ruby on Rails or Node JS, plus any back-end technologies under the right circumstances! After all, you've spent time working with APIs from the front-end already!
+Completing the JavaScript course is a major milestone! Congratulations! You've learned a lot of major JavaScript concepts like prototypes, closures, promises, and event loops. Take a moment to reflect on how far you've come.
 
-However, there is still more to go. If you're following the Ruby on Rails path, there is one more section, Getting Hired, with excellent content on finding a developer job. As you continue forward, keep an attitude of learning through building great software and always exploring more on your own. It will carry you far.
+At this point, you can create phenomenal and dynamic frontends, all using vanilla JavaScript. You've spent time working with APIs and should be confident enough to integrate your frontends to any backends you wish.   
```

Comment:
if the React course will still be right after battleship, this will not be true. I really don't know how to integrate my frontends to any backends yet.